include/envoy/api/os_sys_calls.h

```diff
@@ -155,6 +155,11 @@ class OsSysCalls {
    * @see man 2 write
    */
   virtual SysCallSizeResult write(os_fd_t socket, const void* buffer, size_t length) PURE;
+
+  /**
+   * @see man 2 accept
```

Comment:
We may want to say something about it also marking the socket as non-blocking.

Reply:
Good point. Added!